_posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: blog-post
+title: Announcing Ballerina 2201.7.0 (Swan Lake Update 7)
+author: Ballerina Team
+published-date: 7 July 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are excited to announce the [Ballerina 2201.7.0 (Swan Lake Update 7)](https://ballerina.io/downloads/) release, which adds many new features and improvements to the Ballerina programming language. 
+
+The two main highlights of this release are the support for generating GraalVM native executables and adding aggregation and grouping capabilities to Ballerina query expressions and query actions, which are described below.
+
+## Generation of GraalVM native executables
+
+The Swan Lake Update 7 release comes with official support for generating GraalVM native executables for Ballerina. This new feature allows you to compile Ballerina programs into standalone native executables.
```

Comment:
Can we avoid say `new feature` as we had this already as experimental in previous versions? @sameerajayasoma / @shafreenAnfar

Reply:
+1

Reply:
+1

Reply:
Fixed via [5e638e2](https://github.com/ballerina-platform/blog/pull/195/commits/5e638e2773579b0372d82cca621b762f3090e919)